crates/volta-core/src/hook/mod.rs

```diff
@@ -173,31 +173,20 @@ impl HookConfig {
                 file: file_path.to_path_buf(),
             })?;
 
-        let hooks_path = file_path.parent().unwrap_or_else(|| Path::new("/"));
+        // Invariant: Since we successfully loaded the file, file_path _must_ have a parent
```

Comment:
We could use `expect`, however in this case the invariant is that we definitely loaded a _file_ from that path earlier in the execution. The `.parent()` of a file path is the directory that file is contained in, so even if we were loading `/hooks.json`, the path would still have a parent property.

There aren't any file systems I'm aware of on the supported platforms that allow for files to be top-level (not contained in a directory at all), so the fact that we know it's a file (rather than a directory) means that we can safely unwrap `.parent()` here.

Reply:
Ah, perfect. I thought that was the case, but I couldn’t remember, so GOOD. Thanks for clarifying!